roadservice/core/models.py

```diff
@@ -233,13 +293,93 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.type = Role.Type.COUNTY_MODERATOR
 
-    # TODO
-    def assign_county_expert(self):
-        pass
+    @property
+    def county(self):
+        return self.region.get_concrete()
 
     def get_concrete(self):
         return self
 
+    def add_service_team(self, speciality, team_members_users):
+        for team_member in team_members_users:
```

Comment:
When we use the term `member` we're referring to a foreign key to `Serviceman`. It's better to use the term `user` here.